lib/faye/redis.rb

```diff
@@ -208,6 +214,10 @@ def register_connection_listeners(name, connection, connection_name)
       end
     end
 
+    def configure_inactivity_check(connection, inactivity_check)
+      connection.configure_inactivity_check(inactivity_check[:trigger_secs], inactivity_check[:response_timeout])
```

Comment:
This was added to `em-hiredis` 0.3.1 (https://github.com/mloughran/em-hiredis/commit/eda484fac6211bb6bf14475f104da01b03c8af18).

So we should either bump the gemspec to 0.3.1 or ensure that the method is defined before calling it (and probably log if it isn't).

This is somewhat pedantic, but I don't want us to lose time debugging if we're using an older version of the gem somewhere.

Reply:
I overlooked this. I think it's best to bump the gemspec.